pkg/sentry/inet/inet.go

```diff
@@ -153,3 +153,21 @@ type Route struct {
 	// GatewayAddr is the route gateway address (RTA_GATEWAY).
 	GatewayAddr []byte
 }
+
+// Describes Ip line of /proc/net/snmp.
```

Comment:
This comment should start with the name of the thing being described.
// StatSnmpIp describes the Ip line of /proc/net/snmp.
type StatSnmpIp [19]uint64

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Done. Thanks!